app/assets/stylesheets/v3/layout_components/siteheader.scss

```diff
@@ -81,7 +85,7 @@
 }
 
 .top-nav-offset {
-  margin-top: $top-bar-height;
+  margin-top: $top-bar-height + $side-nav-height;
```

Comment:
on mobile, this seems a bit tall.

Reply:
I'll take a look!

Reply:
You're right, I've updated this for mobile.